app/src/main/java/com/sdn/teampredators/polima/ui/aspirant/AspirantFragment.kt

```diff
@@ -0,0 +1,93 @@
+package com.sdn.teampredators.polima.ui.aspirant
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.View
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import com.sdn.teampredators.polima.R
+import com.sdn.teampredators.polima.databinding.FragmentAspirantBinding
+import com.sdn.teampredators.polima.ui.home.model.Politician
+import com.sdn.teampredators.polima.utils.GenericActions
+import com.sdn.teampredators.polima.utils.load
+import com.sdn.teampredators.polima.utils.viewBinding
+import com.sdn.teampredators.polima.utils.viewState
+import kotlinx.coroutines.delay
+
+class AspirantFragment : Fragment(R.layout.fragment_aspirant), ToAspirantThreeScreens {
```

Comment:
Why are you mixing architectures? We are using MVVM not MVP, stop adding MVP features to the project. The `ToAspirantThreeScreens` interface isn't necessary, you could have handled everything the way we've been handling it